helper/configure-image.sh

```diff
@@ -1,20 +1,23 @@
 #!/usr/bin/env sh
-# Do not us bash. Not all containers have bash installed
-set -ex
+# Use a generic shell instead of bash for compatibility across different containers
 
-# Check configuration
-[ -z "$DDE_UID" ] && echo "DDE_UID is not set" && exit 1;
-[ -z "$DDE_GID" ] && echo "DDE_GID is not set" && exit 1;
+set -ex
+# -e: Exit immediately if a command exits with a non-zero status
+# -x: Print commands and their arguments as they are executed
 
-###############################################################################
+# Check if necessary environment variables are set
+[ -z "$DDE_UID" ] && echo "DDE_UID is not set" && exit 1
+[ -z "$DDE_GID" ] && echo "DDE_GID is not set" && exit 1
 
+# Function to check if a command exists in the PATH
 commandExists() {
   command -v "$1" >/dev/null 2>&1
 }
 
-###############################################################################
-
-# Check plaform
+DEFAULT_SHELL="sb"
+# Set default shell to /bin/sb, can be overridden by setting DDE_SHELL
+SHELL_TO_INSTALL="/bin/${DDE_CONTAINER_SHELL:-$DEFAULT_SHELL}"
```

Comment:
@sbaerlocher how does DDE_CONTAINER_SHELL get into the container? If a configuration/image change is needed it should be documented